fdb-extensions/src/main/java/com/apple/foundationdb/async/MoreAsyncUtil.java

```diff
@@ -912,6 +912,32 @@ public static void closeIterator(@Nonnull Iterator<?> iterator) {
         }
     }
 
+    /**
+     * A utility to close multiple {@link AutoCloseable} objects, preserving all the caught exceptions.
+     * The method would attempt to close all closeables in order, even if some failed.
+     * @param closeables the given sequence of {@link AutoCloseable}
+     * @throws CloseException in case any exception was caught during the process. The first exception will be added
+     * as a {@code cause}. In case more than one exception was caught, it will be added as Suppressed.
+     */
+    @SuppressWarnings("PMD.CloseResource")
+    public static void closeAll(AutoCloseable... closeables) throws CloseException {
```

Comment:
`MoreAsyncUtil` does not seem like the appropriate place for this, perhaps a new `CloseableUtils`.

Reply:
Done.